### PR TITLE
Saransh Duggal | WARG Autonomy Onboarding Submission

### PR DESCRIPTION
**Pull request by saranshrajivduggal**
Completed CIFAR-10 Image Classifier using the ResNet-18 model. All plots, along with training, validation and testing accuracies are within the WARG_Bootcamp.ipynb file. This classifier was trained over 30 epochs.

### CODE REVIEW

**Review by mgupta27 — CHANGES_REQUESTED** [review]
For some reason the `.ipynb` file doesn't show up correctly. Can you fix this and then request for me to review?

**Review by mgupta27 — CHANGES_REQUESTED** [review]
* The only transform that should be applied to the testing dataset is `toTensor`. You don't want to make any changes to the test dataset as you want to test it on unmodified data.
* I don't think it's a good idea to add the `RandomCrop` for training data since the model will be trained on image data with padding when the test data will be image data with no padding.

Rest of the code looks good. I would just make those changes to the transforms and then I will approve the PR. No need to train again as it takes a while and you surpassed 70%.

As for next steps make sure to read [Autonomy Onboarding](https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/2060353537/Autonomy+Onboarding)

**Review by mgupta27 — APPROVED** [review]
LGTM. Look through the [Autonomy Onboarding page](https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/2060353537/Autonomy+Onboarding) for next steps.
